Formula/sail.rb

```diff
@@ -47,6 +47,11 @@ class Sail < Formula
     sha256 "b8aa58f8cf793ffd8782d3d8cb19e66ef36f7aba4353eec859e74678b01b07a7"
   end
 
+  resource "sailed.io" do
+    url "https://files.pythonhosted.org/packages/e2/1c/9c2dc6d5ce5ca8017f1d40271999c7f11e44cbcbd8a6fb68b965284a6dd3/sailed.io-0.9.10.tar.gz"
+    sha256 "d54a504ca6aa8c175727507737ef0c8ceeee1e38cfc5742bc1561b158f13f1ba"
+  end
```

Comment:
How did the source tarball get added as a resource via `brew bump-formula-pr`?

Reply:
the formula is named `sail`, but the resource is named `sailed.io` 🤦